### ISSUE TITLE
Fix fluentd GCS support

### ISSUE THREAD

**Issue by Ajarmar** [elastisys/compliantkubernetes-apps#243]
**Is your feature request related to a problem? Please describe.**

Fluentd currently does not work when using GCS as object storage because of a missing output plugin. (It worked previously with the `stable/fluentd` chart, but not with the new `bitnami/fluentd` chart due to the two being set up in different ways.) The issue occurs because [these plugins](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/fluentd-configmap.yaml.gotmpl#L78) need to be installed, but they are currently not.

**Describe the solution you'd like**

The best solution would be for bitnami's fluentd chart to become a bit more flexible with adding custom plugins. [They seem open to this improvement, see this comment (which also contains some pointers for how it would be done)](https://github.com/bitnami/charts/issues/2941#issuecomment-651255527)

If this can be upstreamed (to [this repository,](https://github.com/bitnami/bitnami-docker-fluentd) not the one the above issue is submitted in) then our [fluentd aggregator ConfigMap](https://github.com/elastisys/compliantkubernetes-apps/blob/main/helmfile/values/fluentd-configmap.yaml.gotmpl) would just need to be updated to set the appropriate environment variables specifying which plugins should be installed.

**Describe alternatives you've considered**

[According to bitnami's documentation,](https://github.com/bitnami/bitnami-docker-fluentd#extend-this-image) adding custom plugins currently requires extending their image. As such, an alternative solution would be:

1. Extend bitnami's fluentd image by writing a new Dockerfile (adding the `digest-crc` and `fluent-plugin-gcs` plugin), perhaps in a new repository of ours
2. Build the image and upload it to our [dockerhub](https://hub.docker.com/u/elastisys)
3. Add a fluentd chart to compliantkubernetes-apps (in `helmfile/charts/`) that uses this new image

**Additional context**

**Definition of done:**
When running `bin/ck8s apply sc` with GCS as object storage, fluentd is deployed and works properly without crashlooping.

**Comment by cristiklein** [comment]
This is something we should definitely do once demand and operational overhead justifies it. However, for now, using [MinIO as a gateway to GCS](https://docs.min.io/docs/minio-gateway-for-gcs.html) feels just the right amount of engineering. MinIO, when used as a gateway, is stateless and non-persistent, and tends to not add headaches.

**Comment by Ajarmar** [comment]
If MinIO will be our "official" solution (for now) for making fluentd work on Google Cloud, would it be appropriate to make an issue for testing setting it up ( + writing documentation on how to set it up)? I don't know how easy it is to set up, but it seems like a good idea to me

**Comment by cristiklein** [comment]
@Ajarmar Yes, please create one. This would be good both for Azure and GCP support.
